src/test/java/chambre/morsecode/WordleGameTest.java

```diff
@@ -0,0 +1,30 @@
+package chambre.morsecode;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Collections;
+
+import static chambre.morsecode.CharResult.Correct;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.mockito.Mockito.doReturn;
+
+class WordleGameTest {
+    @Test
+    public void guess() throws FileNotFoundException {
```

Comment:
Write a test for incorrect 
Also this test doesn't need the throws clause anymore